ldes-server-port-ingest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/extractor/MemberExtractorDecorator.java

```diff
@@ -0,0 +1,19 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.ingest.extractor;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.ingest.entities.IngestedMember;
+import org.apache.jena.rdf.model.Model;
+
+import java.util.List;
+
+public abstract class MemberExtractorDecorator implements MemberExtractor {
+	private final MemberExtractor baseMemberExtractor;
+
+	protected MemberExtractorDecorator(MemberExtractor baseMemberExtractor) {
+		this.baseMemberExtractor = baseMemberExtractor;
+	}
+
+	@Override
+	public List<IngestedMember> extractMembers(Model ingestedModel) {
+		return baseMemberExtractor.extractMembers(ingestedModel);
+	}
+}
```

Comment:
Would strongly advise against this. An abstract class that is just used in one class (just to avoid calling baseMemberExtractor.extractMembers(ingestedModel)) looks very overkill